pscheduler-test-s3throughput/s3throughput/cli-to-spec

```diff
@@ -127,7 +130,10 @@ else:
     result['bucket'] = 'pscheduler-test'
 
 if options.secret_key is not None:
-    result['secret-key'] = options.secret_key
+    if options.secret_key[0] == '@':
```

Comment:
No need to do this check.  `string_from_file()` will do it for you and return the string unaltered if it didn't start with an `@`.  Same at line 120.

Reply:
ok great!